packages/pds/src/account-manager/helpers/account.ts

```diff
@@ -64,6 +64,41 @@ export const getAccount = async (
   return found || null
 }
 
+export const getAccounts = async (
+  db: AccountDb,
+  handlesAndDids: string[],
```

Comment:
also kysely gets mad if you pass in an empty array, so the first thing I'd do in this function is check if the array length is 0 & if so then just return an empty map